src/ui/ui_get_public_key.c

```diff
@@ -106,14 +113,14 @@ UX_DEF(ux_compare_pk_flow, &ux_compare_pk_flow_1_step);
 
 static void compare_pk() { ux_flow_init(0, ux_compare_pk_flow, NULL); }
 
-unsigned int io_seproxyhal_touch_pk_ok(const bagl_element_t *e) {
+static unsigned int io_seproxyhal_touch_pk_ok(const bagl_element_t *e) {
```

Comment:
Not related to stax porting, but maybe you can go a bit farther by rename this function with something more clear and removing the unused parameter?

Reply:
Done